src/App.php

```diff
@@ -24,9 +24,9 @@
 namespace Facebook;
 
 use Facebook\Authentication\AccessToken;
-use Facebook\Exceptions\FacebookSDKException;
+use Facebook\Exceptions\SDKException;
 
-class FacebookApp implements \Serializable
+class App implements \Serializable
```

Comment:
as this PR is about renaming, what about naming this one `Application`?

Reply:
Thank you